"could not load module" errors at startup

Thank you, this is awesome!

But I hit trouble while following correct (I believe) install procedure, on a mac:
```
...
GHCi, version 8.8.3: https://www.haskell.org/ghc/  :? for help
Loaded package environment from /Users/simon/.ghc/x86_64-darwin-8.8.3/environments/default
Warning: -debug, -threaded and -ticky are ignored by GHCi
package flags have changed, resetting and loading new packages...
package flags have changed, resetting and loading new packages...
Loaded GHCi configuration from /Users/simon/src/blog/.ghci
Loaded GHCi configuration from /Users/simon/.ghci
Prelude> package flags have changed, resetting and loading new packages...
package flags have changed, resetting and loading new packages...

<no location info>: error:
    Could not load module ‘ClassyPrelude’
    It is a member of the hidden package ‘classy-prelude-1.5.0’.
    You can run ‘:set -package classy-prelude’ to expose it.
    (Note: this unloads all the modules in the current scope.)
Prelude> Prelude> [1 of 6] Compiling Application.Helper.Controller ( Application/Helper/Controller.hs, interpreted )
[2 of 6] Compiling Config           ( Config/Config.hs, interpreted )

[3 of 6] Compiling Generated.Types  ( build/Generated/Types.hs, interpreted )
build/Generated/Types.hs:7:1: error:
    Could not load module ‘CorePrelude’
    It is a member of the hidden package ‘basic-prelude-0.7.0’.
    You can run ‘:set -package basic-prelude’ to expose it.
    (Note: this unloads all the modules in the current scope.)
Failed, two modules loaded.
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
  |
7 | import CorePrelude hiding (id) 
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
...
```

--- a/exe/IHP/IDE/DevServer.hs
+++ b/exe/IHP/IDE/DevServer.hs
@@ -38,7 +38,7 @@ main = do
 
     -- Print IHP Version when in debug mode
     when isDebugMode (putStrLn ("IHP Version: " <> Version.ihpVersion))
-    
+
     let ?context = Context { actionVar, portConfig, appStateRef, isDebugMode }
 
     threadId <- myThreadId
@@ -67,7 +67,7 @@ handleAction state@(AppState { statusServerState = StatusServerNotStarted }) (Up
 handleAction state@(AppState { statusServerState = StatusServerStarted { } }) (UpdateStatusServerState StatusServerNotStarted) = pure state { statusServerState = StatusServerNotStarted }
 handleAction state@(AppState { statusServerState = StatusServerPaused { } }) (UpdateStatusServerState statusServerState) = pure state { statusServerState = StatusServerNotStarted }
 handleAction state@(AppState { liveReloadNotificationServerState = LiveReloadNotificationServerNotStarted }) (UpdateLiveReloadNotificationServerState liveReloadNotificationServerState) = pure state { liveReloadNotificationServerState }
-handleAction state@(AppState { liveReloadNotificationServerState = LiveReloadNotificationServerStarted {} }) (UpdateLiveReloadNotificationServerState liveReloadNotificationServerState) = 
+handleAction state@(AppState { liveReloadNotificationServerState = LiveReloadNotificationServerStarted {} }) (UpdateLiveReloadNotificationServerState liveReloadNotificationServerState) =
     case liveReloadNotificationServerState of
         LiveReloadNotificationServerNotStarted -> pure state { liveReloadNotificationServerState }
         otherwise -> error "Cannot start live reload notification server twice"
@@ -86,7 +86,7 @@ handleAction state@(AppState { appGHCIState, statusServerState, postgresState })
             let statusServerState' = case statusServerState of
                     StatusServerStarted { .. } -> StatusServerPaused { .. }
                     _ -> statusServerState
-            
+
             pure state { appGHCIState = appGHCIState', statusServerState = statusServerState' }
         RunningAppGHCI { } -> pure state -- Do nothing as app is already in running state
         AppGHCINotStarted -> error "Unreachable AppGHCINotStarted"
@@ -107,7 +107,7 @@ handleAction state@(AppState { appGHCIState, statusServerState, postgresState, l
                 AppGHCIModulesLoaded { .. } -> AppGHCIModulesLoaded { .. }
                 RunningAppGHCI { .. } -> AppGHCIModulesLoaded { .. }
                 AppGHCINotStarted {} -> error "Modules cannot be loaded when ghci not in started state"
-    
+
     notifyHaskellChange liveReloadNotificationServerState
 
     pure state { statusServerState = statusServerState', appGHCIState = newAppGHCIState }
@@ -118,7 +118,7 @@ handleAction state@(AppState { statusServerState, appGHCIState, liveReloadNotifi
         AppGHCIModulesLoaded { .. } -> pure state { appGHCIState = RunningAppGHCI { .. } }
         RunningAppGHCI { } -> pure state
         otherwise -> pure state
-    
+
 handleAction state@(AppState { liveReloadNotificationServerState }) AssetChanged = do
     notifyAssetChange liveReloadNotificationServerState
     pure state
@@ -133,7 +133,7 @@ handleAction state@(AppState { liveReloadNotificationServerState, appGHCIState, 
 
     clearStatusServer statusServerState
 
-    let appGHCIState' = 
+    let appGHCIState' =
             case appGHCIState of
                 AppGHCILoading { .. } -> AppGHCILoading { .. }
                 AppGHCIModulesLoaded { .. } -> AppGHCILoading { .. }
@@ -218,6 +218,7 @@ startGHCI = do
             , "-fomit-interface-pragmas"
             , "-j"
             , "-O0"
+            , "-package-env -" -- Do not load `~/.ghc/arch-os-version/environments/name file`, global packages interfere with our packages
             , "-ignore-dot-ghci" -- Ignore the global ~/.ghc/ghci.conf That file sometimes causes trouble (specifically `:set +c +s`)
             , "-ghci-script", ".ghci" -- Because the previous line ignored default ghci config file locations, we have to manual load our .ghci
             , "+RTS", "-A512m", "-n4m", "-H512m", "-G3", "-qg", "-N"
@@ -244,7 +245,7 @@ startAppGHCI = do
 
     libDirectory <- LibDir.findLibDirectory
 
-    let loadAppCommands = 
+    let loadAppCommands =
             [ ":script " <> cs libDirectory <> "/applicationGhciConfig"
             , ":set prompt \"\"" -- Disable the prompt as this caused output such as '[38;5;208mIHP>[m Ser[v3e8r; 5s;t2a0r8tmedI' instead of 'Server started'
             , "import qualified ClassyPrelude"

--- a/lib/IHP/Makefile.dist
+++ b/lib/IHP/Makefile.dist
@@ -2,7 +2,7 @@
 
 # Use bash as we sometimes use bash specific syntax, like pipefail below
 #SHELL=/bin/bash
-#.SHELLFLAGS := -eu -o pipefail -c  
+#.SHELLFLAGS := -eu -o pipefail -c
 .ONESHELL:
 
 MAKEFLAGS += --warn-undefined-variables
@@ -62,6 +62,7 @@ GHC_RTS_FLAGS := -A256m -n2m -N
 GHC_OPTIONS=-threaded -i. -ibuild -iConfig -iIHP
 GHC_OPTIONS+= ${GHC_EXTENSIONS}
 GHC_OPTIONS+= -j2
+GHC_OPTIONS+= -package-env - # https://github.com/digitallyinduced/ihp/issues/130
 
 ifneq ($(GHC_RTS_FLAGS),)
 GHC_OPTIONS+= +RTS ${GHC_RTS_FLAGS} --RTS

